### PR TITLE
chore: Use NPM CI for CI

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -106,6 +106,6 @@ jobs:
         uses: actions/setup-node@v2
       - name: Npm Install
         if: steps.cache.outputs.cache-hit != 'true'
-        run: npm install
+        run: npm ci
       - name: Run diff check for the breakUpAriaJSON script
-        run: node scripts/breakUpAriaJSON.js && git diff --exit-code -- src
+        run: node scripts/breakUpAriaJSON.js && git diff --exit-code
```
Comment (jessebeach) [inline .github/workflows/test.yml:111]: @nschonni can you revert this change? The diff check should only happen on the src dir, not the whole repo

Comment (jessebeach) [reply .github/workflows/test.yml:111]: Just the one change revert.

**Review by jessebeach — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (nschonni) [reply .github/workflows/test.yml:111]: I can revert this, but this was only required because you weren't using `ci` for the install before. The lockfile won't be changing any more

Comment (jessebeach) [reply .github/workflows/test.yml:111]: It performs the intended check explicitly, so no harm here.

**Review by jessebeach — APPROVED** [review]
